README.md

```diff
@@ -99,6 +99,12 @@ To run tests after setting up the linters, run the following command:
 - Twitter: [@gmail](email2faizm@gmail.com)
```

Comment:
- [ ] This here should be your account for `X` not your gmail account, please kindly correct this :+1: